spec/rails_helper.rb

```diff
@@ -0,0 +1,63 @@
+# This file is copied to spec/ when you run 'rails generate rspec:install'
+require "spec_helper"
+ENV["RAILS_ENV"] ||= "test"
+require_relative "../config/environment"
+# Prevent database truncation if the environment is production
+abort("The Rails environment is running in production mode!") if Rails.env.production?
+require "rspec/rails"
+# Add additional requires below this line. Rails is not loaded until this point!
+
+# Requires supporting ruby files with custom matchers and macros, etc, in
+# spec/support/ and its subdirectories. Files matching `spec/**/*_spec.rb` are
+# run as spec files by default. This means that files in spec/support that end
+# in _spec.rb will both be required and run as specs, causing the specs to be
+# run twice. It is recommended that you do not name files matching this glob to
+# end with _spec.rb. You can configure this pattern with the --pattern
+# option on the command line or in ~/.rspec, .rspec or `.rspec-local`.
+#
+# The following line is provided for convenience purposes. It has the downside
+# of increasing the boot-up time by auto-requiring all files in the support
+# directory. Alternatively, in the individual `*_spec.rb` files, manually
+# require only the support files necessary.
+#
+# Dir[Rails.root.join('spec', 'support', '**', '*.rb')].sort.each { |f| require f }
+
+# Checks for pending migrations and applies them before tests are run.
+# If you are not using ActiveRecord, you can remove these lines.
+begin
+  ActiveRecord::Migration.maintain_test_schema!
+rescue ActiveRecord::PendingMigrationError => e
+  abort e.to_s.strip
+end
+RSpec.configure do |config|
+  # Remove this line if you're not using ActiveRecord or ActiveRecord fixtures
+  config.fixture_path = "#{::Rails.root}/spec/fixtures"
```

Comment:
Nice job @duleorlovic.
Do we have now fixtures? Can I seed for example users with them?

Reply:
Yes, I'm waiting for this to be approved and merged and I will create fixtures.

Reply:
Sorry, I am not sure if the fixture works. It was a nice job so far, but try to make one fast test using fixtures.
I need to be sure that the configuration is ok without warnings.
Also, try to seed Users with fixtures.
Happy coding.

Reply:
@DeVuDeveloper I need this merged in order to submit seed example, can you approve ?

Reply:
Hi, @duleorlovicj👋,

# STATUS: APPROVED 🕺🥳 

Congratulations🍾🥳!!   There is nothing else to say other than... it's time to merge it :shipit: 

![](https://media.giphy.com/media/ygx6guQZ3kxF8A80bC/giphy.gif)

Good luck! 🎉